### PR TITLE
Insert db for issue-10

### PR DESCRIPTION
**Pull request by ledenev-viktor**
for [issue-10](https://github.com/ledenev-viktor/music-studio/issues/10)

### CODE REVIEW

File: src/components/widgets/registration/steps/ContactsStep.tsx [hunk 1]
```diff
@@ -13,13 +14,31 @@ export const ContactsStep = ({
...
+    const { clearErrors, control } = useFormContext();
+
+    const fieldUserNameTelegram = useWatch({
+        control,
+        name: 'userNameTelegram',
+    });
+    const fieldUserNameInstagram = useWatch({
+        control,
+        name: 'userNameInstagram',
+    });
```
Comment (YuliaKletsova) [inline src/components/widgets/registration/steps/ContactsStep.tsx:26]:
```suggestion
    const [fieldUserNameTelegram, fieldUserNameInstagram] = useWatch< FormFields , ['userNameTelegram', 'useNameInstagram']>({
        name: ['userNameTelegram', 'useNameInstagram'],
    });
```

also i would call it shorter userNameTelegram and useNameInstagram

File: src/components/widgets/registration/useGetReviewInfo.tsx [hunk 2]
```diff
@@ -93,7 +94,14 @@ export const useGetReviewInfo = (
                     onEditClick={() => onClick(STEP_TYPE.USER_INFO)}
                 >
                     <Typography.Paragraph>
-                        Telegram: {userNameTelegram}
+                        <div>
+                            Telegram:{' '}
```
Comment (YuliaKletsova) [inline src/components/widgets/registration/useGetReviewInfo.tsx:98]: Well)) this is a wrong usage for Typograpy.Paragraph. Just create to tags and make them under Flex. Also, you have conditions, but fields are required right?

**Review by YuliaKletsova — CHANGES_REQUESTED** [review]

File: src/components/widgets/registration/useGetReviewInfo.tsx (hunk 2 above)
Comment (ledenev-viktor) [reply src/components/widgets/registration/useGetReviewInfo.tsx:98]: >Also, you have conditions, but fields are required right?

Not entirely true, these fields are conditional. One contact out of two required

**Review by YuliaKletsova — APPROVED** [review]
LGTM

**Pull request merged**